src/taskgraph/util/dependencies.py

```diff
@@ -35,7 +34,7 @@ def group_by_all(config, tasks):
     return [[task for task in tasks]]
 
 
-@group_by("attribute", schema=Schema(str))
+@group_by("attribute")
```

Comment:
I guess there's no easy way to validate a schema with no keys with msgspec? Seems fair enough to just drop the schema here then